app/src/main/kotlin/com/wire/android/ui/home/conversations/search/SearchPeopleRouter.kt

```diff
@@ -178,14 +178,6 @@ fun SearchUsersAndServicesScreen(
                                 }
                             }
                         }
-
-                        LaunchedEffect(pagerState.isScrollInProgress, focusedTabIndex, pagerState.currentPage) {
-                            if (!pagerState.isScrollInProgress && focusedTabIndex != pagerState.currentPage) {
-                                keyboardController?.hide()
-                                focusManager.clearFocus()
-                                focusedTabIndex = pagerState.currentPage
-                            }
-                        }
```

Comment:
Doesn't this also handles when a different tab is selected (like the tabs on the top : PEOPLE | SERVICES) ?

Reply:
nope the tab changes is handled already somewhere else.